src/System.Memory/tests/Span/CopyTo.cs

```diff
@@ -204,6 +204,12 @@ public static void CopyToCovariantArray()
         [InlineData((4L * 1024L * 1024L * 1024L) + 256)]
         public static void CopyToLargeSizeTest(long bufferSize)
         {
+            // If this test is run in a 32-bit process, the large allocation will fail.
+            if (Unsafe.SizeOf<IntPtr>() != sizeof(long))
```

Comment:
> What is the difference between Unsafe.SizeOf<>() and sizeof()?

`sizeof(IntPtr)` can only be called from an "unsafe" context. `Unsafe.SizeOf<T>()` can be called anywhere. They are otherwise identical. IntPtr.Size will also give you the same value.